docs/guidance/map/framework_agreements.md

```diff
@@ -81,23 +81,24 @@ The two extensions should be declared in the package metadata:
 :language: json
 ```
 
-The following guidance describes how to model the different stages of a framework agreement procedure in OCDS.  
+The following guidance describes how to model the different stages of a framework agreement procedure in OCDS. (The guidance does cover modelling that is the same as for other types of contracts, for example populating `contracts`). 
```

Comment:
Yes, missing, sorry, added.

Modeling - oops, ok, sorry, fixed. 

"Other types of procedures" - not really. Frameworks are a particular type of contract, so "other types of contracts" was the intended formulation. To me it doesn't seem confusing, but I understand that to other it might. If you think that is the case, we can think of a formulation that avoids mentioning contracts? 

For reference, here is the definition of contract 

> "Information regarding the contract, typically between the buyer and supplier. This includes contracts describing all the contractual conditions (e.g. item, quantity, price, payment terms, time and place of delivery), as well as contracts only describing the general contractual conditions (such as a framework agreement) and those only describing the specific contractual conditions (such as a contract within a framework agreement). Communication between contractual parties that consists of minor specifications of conditions agreed previously (e.g. specifying the time or place of delivery) is not considered a contract. Amendments are considered as part of the contract that is being amended. Contracts are also used when giving prizes or other rewards (e.g. a follow-up contract) resulting from a design contest."

Reply:
Aha, maybe then this?

> This guidance doesn't cover the modeling of the contract (e.g. via the `contracts` array), because these details of a framework agreement are modeled the same as for other types of contracts.

Reply:
Sounds good! Done in https://github.com/open-contracting/standard/pull/1461/commits/bbf9bf21b19b685315c5e6b21023835a994d1d70.